### DISCUSSION TITLE
Removing elements from collection pages

### DISCUSSION THREAD

**Discussion by mjh2901** [mmistakes/minimal-mistakes#3883] [Q&A]
I am using the collection pages in a collection
https://disneylandian.com/dr/dca_grizzly_flats/
I am trying to remove the Updated date and the arrows for going to the next page.
I do not want to edit the actual template, I was hoping there was a command to place in the config, sadly I have tried a bunch of options from looking at the instructions (which are awesome) and can't figure it out.

Anyone have any ideas? 

This is the config
_config.yml
  - scope:
      path: ""
      type: dr
    values:
      layout: single
      author_profile: false
      read_time: false
      comments: gtrue
      share: false
      related: true

Actual page header
title: "DCA Grizzly Peak Airfield"
excerpt: "Music Loop."
collection: "dr"
header:
  overlay_image: /assets/images/dca_grizzly_flats.jpg
  overlay_filter: 0.5 # same as adding an opacity of 0.5 to a black background
  #caption: "Photo credit: [**Unsplash**](https://unsplash.com)"
  teaser: /assets/images/dca_grizzly_flats.jpg
  actions:
    #- label: "More Info"
    #  url: "https://unsplash.com"

**Comment by iBug** [comment]
> I do not want to edit the actual template

You have just eliminated the only answer.

**Reply by mjh2901** [reply]
Dam, I was hoping there was a config switch and I was missing it in the documentation...  So many parts have true and false options.  I will have to look into how to create a local template file since this is a linked site.
